client/src/components/ToppingsCarousel.tsx

```diff
@@ -17,25 +16,25 @@ function ToppingsCarousel() {
     const [toppingImgUrls, setToppingImgUrls] = useState([Caramel, Marshmallow, Strawberry, Chocolate, Nuts, Sprinkles]);
     function rotateArr(direction: string): void {
         // don't mutate original array. make a copy then mutate the copy
-        let newArr: string[] = toppingImgUrls.slice();
+        const newArr: string[] = toppingImgUrls.slice();
         if (newArr.length > 1) {
             if (direction == 'left') {
                 // .shift() removes and returns the first element of the array.
                 // .push() pushes the value onto the end of the array.
-                let x = newArr.shift();
-                if (x) newArr.push(x);
+                const topping = newArr.shift();
+                if (topping) newArr.push(topping);
             } else {
                 // .pop() removes and returns the last element of the array.
                 // .unshift() 'pushes' the value onto the beginning of the array.
```

Comment:
Are these comments necessary here?

Reply:
Definitely not! I was just hesitant to clean up someone else's file. Will do now.